debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/UniqueDatabase.java

```diff
@@ -169,6 +169,7 @@ public UniqueDatabase withDbHistoryPath(final Path dbHistoryPath) {
                 .with(MySqlConnectorConfig.DATABASE_WHITELIST, getDatabaseName())
                 .with(MySqlConnectorConfig.DATABASE_HISTORY, FileDatabaseHistory.class)
                 .with(MySqlConnectorConfig.BUFFER_SIZE_FOR_BINLOG_READER, 10_000);
+//        TODO rkuchar create a new profile to run the test with LEGACY ddl parser
```

Comment:
Do you still plan to do this? WDYM by "profile", a Maven profile?

Reply:
Yes that was meant for a maven profile, but I have little experience with creating a profile and using properties in them. I'm not able to run integration test locally, because the maven will freeze when it is starting docker container with database, so i was not able to test this new profile. I don't want to commit untested code in which i have no idea if it is good or not, so i was hoping that some of you could handle this if there is actually need for it :).

Reply:
No problem, filed DBZ-734 for that.